tools/cdk-build-tools/config/eslintrc.js

```diff
@@ -193,5 +195,12 @@ module.exports = {
         'method',
       ],
     }],
+
+    // Overrides for plugin:jest/recommended
+    "jest/no-conditional-expect": "warn",
```

Comment:
I've found warns to be useless. Either off or error.

Reply:
:shrug: To each their own. I'd love a second opinion. Personally, when I run a build on a package -- especially one that I "own" -- and see warnings jumping out, I'm going to at least look at them and see if there's a quick re-write to fix it.

`jest/no-conditional-expect` - A fairly weird one; this complains when you have `expects` inside of a conditional block. I looked at one or two examples, and decided just to mark it as warn because I wasn't sure about the rewrite cost.
`"jest/valid-title"` - Happy to just turn this one off.

Reply:
I guess I'm not too concerned with the above warnings, so turned them both off. Note this doesn't disable _all_ warnings; there are still warnings that exist from the recommended set, and I think many of them are valuable.

Reply:
There are tons of warnings we just ignore. You can find them all if you download a full build output. These range from awslint warnings, eslint warnings, jsii warnings, etc.

The general behaviour seems to be to not care about warnings, and I've adapted my approach to that behaviour. Either it's a problem or it's not.